wazuh/config/data_dirs.env

```diff
@@ -1,9 +1,27 @@
 i=0
-DATA_DIRS[((i++))]="etc"
-DATA_DIRS[((i++))]="ruleset"
+DATA_DIRS[((i++))]="api/configuration"
+DATA_DIRS[((i++))]="etc/client.keys"
+DATA_DIRS[((i++))]="etc/decoders"
+DATA_DIRS[((i++))]="etc/lists"
+DATA_DIRS[((i++))]="etc/local_internal_options.conf"
+DATA_DIRS[((i++))]="etc/localtime"
+DATA_DIRS[((i++))]="etc/ossec.conf"
+DATA_DIRS[((i++))]="etc/ossec-init.conf"
```

Comment:
ossec-init should be not included, isn't it? Otherwise at the moment you upgrade the version won't be updated.